### DISCUSSION TITLE
How to slice a point cloud?

### DISCUSSION THREAD

**Discussion by TianmingQu** [pyvista/pyvista#3468] [Q&A]
Hi all,

I have a point cloud, does anyone know how I can make a slice on this point cloud? And also I would like to do a mesh operation to the point cloud, is there anyway I can do this?

Best,
Tianming

**Comment by banesullivan** [comment]
Slicing is a geometry/dimension-reducing filter. Point clouds (point cells) are 0-dimensional (line cells are 1D, triangles/quads are 2D, and voxels are 3D). When you slice a dataset the geometry/cells are reduced 1 dimension. For example:

- A slice of a voxel (3D) might produce a quad or traingles (2D)
- A slices of a quad or triangle (2D) would produce a line (1D)
- A slice of a line (1D) would produce a point (0D)
- A slice of points (0D) is impossible!

With that in mind, you'll have to increase the dimensionality of your data. One way to do this is by interpolating the values of the point cloud onto a surface or volume. I recommend checking out the examples in https://docs.pyvista.org/examples/01-filter/interpolate.html
